### DISCUSSION TITLE
What is the plan for the language spec?

### DISCUSSION THREAD

**Discussion by yliuuuu** [partiql/partiql-lang#67] [General]
Question ported from community.partiql.org. 
    - @jonwilsdon originally facilitated the Q&A.  

This question was originally asked on Hacker News, and to provide this information to the PartiQL community, I am duplicating it here, lightly edited.

Question

The reference implementation is open source; what’s the plan for the language spec? Is that something that AWS is going to own and control? The website references the PartiQL Steering Committee – is that just AWS folks or is the intention to make it more broadly composed of members of the community you build?

First Answer

The language spec source will be open-sourced, as well, early next week (week of Aug 5, 2019). Overall, we look forward to a community effort and participants that are interested in making significant investments to achieve the project’s goals. Diverse opinions and viewpoints, both on the language and on the process, are very welcome.

At this point, the maintainers/committee is only Amazon members. As PartiQL grows towards a diverse community, we expect to add maintainers/committee (for code and spec) that have non-Amazon affiliations and explore more formalized methods of governance, as they will emerge from our community discussions.

Second Answer

The same question was raised on Twitter, and I put my thoughts there: [https://twitter.com/msw/status/1157405984823758848](https://twitter.com/_msw_/status/1157405984823758848).

TL;DR, my advice is that successful open source projects and open specifications usually have diverse communities. You will have a hard time attracting people to your community of they do not share goals with the rest of the community. We should have some bounding boxes around how the spec evolves through clear tenets. Otherwise welcome diverse opinions, experience, and problems to solve collaboratively.

Source: [original comment and answers on HN](https://news.ycombinator.com/item?id=20594299).

**Comment by yliuuuu** [comment]
Community members are welcomed to contribute to the Spec through our RFC process. See [RFC-template](https://github.com/partiql/partiql-lang/blob/main/RFC-template.adoc) for more details.
